www/about/index.html

```diff
@@ -12,11 +12,8 @@ <h3>Please join us!</h3>
 
             <p>Here's what to do:</p>
             <ol>
-                <li>Sign up for our newsletter to keep up with PyLadies news.</li>
                 <li>Join your local <a href="{{ get_url('/locations') }}">Meetup group</a> or start your own with our <a href="http://github.com/pyladies/pyladies-kit">PyLadies starter kit</a></li>
                 <li>Join our global <a href="https://plus.google.com/communities/108807002736066163985">Google+ community</a> to keep up with recent Pyladies conversations.</li>
```

Comment:
Should this be removed too?

Reply:
Yikes yes let me do that!! Yay for PR review!